src/main/kotlin/org/opensearch/commons/alerting/util/CommonUtilsException.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.commons.alerting.util
+
+import org.apache.logging.log4j.LogManager
+import org.opensearch.OpenSearchException
+import org.opensearch.OpenSearchSecurityException
+import org.opensearch.OpenSearchStatusException
+import org.opensearch.core.common.Strings
+import org.opensearch.core.rest.RestStatus
+import org.opensearch.index.IndexNotFoundException
+import org.opensearch.index.engine.VersionConflictEngineException
+import org.opensearch.indices.InvalidIndexNameException
+
+private val log = LogManager.getLogger(CommonUtilsException::class.java)
+
+class CommonUtilsException(message: String, val status: RestStatus, ex: Exception) : OpenSearchException(message, ex) {
```

Comment:
what benefit are we getting from removing the delegation of execption handling to the plugin? there is no business logic in common utils 
why should we have to throw a common utils exception?

Reply:
Removed `CommonUtilsException` in PR https://github.com/opensearch-project/common-utils/pull/639. Will cherry-pick that commit once that PR is merged.